backend/src/goals/goalsHelper.js

```diff
@@ -69,4 +69,25 @@ const getShortTermGoalsResponseFromDbResult = (result, dayOfWeek) => {
   return responseObj;
 };
 
-module.exports = { getGoalsResponseFromDBResult, getShortTermGoalsResponseFromDbResult }
+
+const updateShortTermGoalCounter = (shortTermGoals, currentShortTermGoals) => {
+  let currentShortTermGoalsMap = new Map();
+
+  currentShortTermGoals.forEach(function(shortTermGoal) {
+    currentShortTermGoalsMap.set(shortTermGoal.title,shortTermGoal.timesCompleted);
```

Comment:
Are we sticking with title here?

Reply:
I don't think i'll have time to update it to ID today because it'll need some schema change. I'll merge it in right now so Lina can test, but I'll update it once I finish with the backend tests.